logger/logger.lua

```diff
@@ -0,0 +1,51 @@
+local string = require('string')
+local windower = require('windower')
+local files = require('files')
+local os = require('os')
+local chat = require('chat')
+local account = require('account')
+
+local get_log = function()
+    local dir = windower.user_path .. '\\' .. account.name .. '\\'
+    os.execute('mkdir "' .. dir .. '" >nul 2>nul')
+
+    local date = os.date('*t')
+    local file_timestamp = string.format('%.4u.%.2u.%.2u.log',date.year, date.month, date.day)
+    
+    return files.create(dir .. file_timestamp)
+end
+
+chat.text_added:register(function(obj)
+    if account.logged_in then
+        local file = get_log()
+        file:append(obj.text, true)
+    end
+end)
+
+--[[
+Copyright © 2018, Windower Dev Team
```

Comment:
I have no real desire to "own" this code in a legal/copyright stance. What should I list instead?

Reply:
It's a little sketchy legally (usually it's done by signing a copyright assignment agreement), but if you want to just hand it over to us... I guess this can work.

Reply:
I decided against a shaky legal precedent to just go with an approach like previous contributions to v4.